app/src/main/java/com/github/se/polyfit/ui/screen/OverviewScreen.kt

```diff
@@ -117,11 +117,16 @@ fun OverviewScreen(
   if (showPictureDialog) {
     PictureDialog(
         onDismiss = { showPictureDialog = false },
-        onFirstButtonClick =
-            overviewViewModel.callCamera(context, startCamera, requestPermissionLauncher),
-        onSecondButtonClick = { pickImageLauncher.launch("image/*") },
-        firstButtonName = context.getString(R.string.take_picture_dialog),
-        secondButtonName = context.getString(R.string.import_picture_dialog))
+        onButtonsClick =
+            listOf(
+                overviewViewModel.callCamera(context, startCamera, requestPermissionLauncher),
+                { pickImageLauncher.launch("image/*") },
+                {}),
```

Comment:
I think it could be better to move this into the OverviewScreen ViewModel as I think it can be considered as Logic

Reply:
I try, but we can't have the pickimageLauncher in the OverviewViewModel. It is either a composable or a fragment or an activity.